lib/mi_bridges/driver/job_income_more_about_page.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class JobIncomeMoreAboutPage < BasePage
+      TITLE = /More About .*'s Job/
+
+      delegate :members, to: :snap_application
+
+      def setup; end
+
+      def fill_in_required_fields
+        fill_in(
+          "Name of Employer",
+          with: current_member.employed_employer_name,
+        )
+
+        select(
+          current_member.employed_pay_interval,
+          from: "How often does #{name} get paid? This is #{name}'s pay period",
+        )
+
+        if hourly?
```

Comment:
I think my comment got swallowed by GitHub before-- if someone is hourly, they are not paid a salary. But if someone is paid weekly, they could still be a non-salaried employee (eg: me!) -- thoughts on how to handle this?

Reply:
I think technically you ARE paid hourly, but bill weekly?

The way through this is to take the word of the applicant. If they say "hourly" we go along with that.

Reply:
yes my point exactly -- having a conditional check if someone is hourly and otherwise assuming a salary seems incorrect?

Reply:
What recourse do we have? Is there another way?

Reply:
If someone is paid yearly, I think we can safely assume they have a salary. Otherwise, we can assume hourly? At least that way we will be reporting their hours per week, which we ask of all ppl but MI Bridges only asks for people who work hourly

Reply:
what do you think of this suggestion?